learning/step_by_step/your_first_game.rst

```diff
@@ -0,0 +1,768 @@
+.. _doc_your_first_game:
+
+Your First Game
+===============
+
+Overview
+--------
+
+This tutorial will guide you through making your first Godot Engine
+project. You will learn how the Godot Engine editor works, how a project
+is structured, and how to build a 2D game.
```

Comment:
Maybe let's follow the writing guidelines for the docs too? direct voice: `how to structure a project`